### PR TITLE
Remove use of netstat in tests

### PR DESCRIPTION
**Pull request by dragon512**
update autest to 1.4.2 as this has fix for portopen to be like netstat, but portable

### CODE REVIEW

**Comment by dragon512** [comment]
[approve ci]

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
[approve ci autest]

**Comment by dragon512** [comment]
Just an update here, it seem the testing is failing because we have two tests that need ATS to be built with brotli. It seems that automake does not see it is install on fedora. I am looking in to why

**Comment by dragon512** [comment]
[approve ci autest]

File: tests/gold_tests/headers/normalize_ae.test.py [hunk 1]
```diff
@@ -98,12 +90,12 @@ def allAEHdrs(shouldWaitForUServer, shouldWaitForTs, ts, host):
 
     if shouldWaitForUServer:
         # wait for the micro server
-        tr.Processes.Default.StartBefore(server, ready=CheckPort(server.Variables.Port))
+        tr.Processes.Default.StartBefore(server)
```
Comment (shinrich) [inline tests/gold_tests/headers/normalize_ae.test.py]: Don't you want to add the WhenPort open check here too to replace the ready=CheckPort?

File: tests/gold_tests/headers/normalize_ae.test.py [hunk 2]
```diff
@@ -98,12 +90,12 @@ def allAEHdrs(shouldWaitForUServer, shouldWaitForTs, ts, host):
...
 
     if shouldWaitForTs:
         # wait for the micro server
         # delay on readiness of port
-        tr.Processes.Default.StartBefore(ts, ready=CheckPort(ts.Variables.port))
+        tr.Processes.Default.StartBefore(ts)
```
Comment (shinrich) [inline tests/gold_tests/headers/normalize_ae.test.py]: Same here?

File: tests/gold_tests/headers/normalize_ae.test.py (hunk 1 above)
Comment (dragon512) [reply tests/gold_tests/headers/normalize_ae.test.py]: no.. This is already set by the testing framework. Part of the fixes here are that we can use the When.PortReady logic and "now" it does it like netstat

Comment (dragon512) [reply tests/gold_tests/headers/normalize_ae.test.py]: Oh .. so we can do another clean up and remove the other port open checks as well. I will do that in a different PR

**Pull request merged**